tests/test_bot.py

```diff
@@ -27,21 +27,10 @@ def bot() -> ModmailBot:
 
     ModmailBot instance.
     """
-    bot: ModmailBot = ModmailBot()
+    bot: ModmailBot = ModmailBot(loop=asyncio.get_event_loop())
```

Comment:
Why are we directly passing the loop here? can't we just run get_event_loop from anywhere? So long as there's no threading.. but there should absolutely be _no threading_.